tarantool_test.go

```diff
@@ -352,6 +356,52 @@ func BenchmarkClientParallelMassiveUntyped(b *testing.B) {
 	close(limit)
 }
 
+func BenchmarkConnection_Replace(b *testing.B) {
+	conn, err := Connect(server, opts)
+	if err != nil {
+		b.Errorf("No connection available")
+		return
+	}
+	defer conn.Close()
+	spaceNo = 520
+
+	b.ResetTimer()
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			_, err := conn.Replace(spaceNo, []interface{}{uint(1), "hello", []interface{}{}})
+			if err != nil {
+				b.Log(err)
+				b.Errorf("No connection available")
+			}
+		}
+	})
+}
+
+func BenchmarkConnection_LargeSelect(b *testing.B) {
+	conn, err := Connect(server, opts)
+	if err != nil {
+		b.Errorf("No connection available")
+		return
+	}
+	defer conn.Close()
+
+	schema := conn.Schema
+	rSpaceNo, rIndexNo, err := schema.ResolveSpaceIndex("test_perf", "secondary")
+	if err != nil {
+		b.Errorf("symbolic space and index params not resolved")
+	}
+
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			_, err := conn.Select(rSpaceNo, rIndexNo, 0, 10000000, IterEq, []interface{}{"test_name"})
```

Comment:
So you select a whole 100000-records space here? I don't get what was the motivation except for checking up the symbol limits of benchstat output
```
BenchmarkConnection_LargeSelect-8         	      27	 115816875 ns/op	212202913 B/op	  699828 allocs/op
```

It would be useful to check selects with response containing more than one tuple, but 100000 of them in one request seems like a bad application design (and with this bench we may give an inspiration to someone to do this).

Reply:
Sounds reasonable, I have changed the count of returning tuples in one response. Now that can be considered as some batch result.